static/docs/user-guide/external-dependencies.md

```diff
@@ -58,3 +59,11 @@ workspace:
                                 hdfs://user@example.com/home/shared/data.txt \
                                 data.txt
 ```
+
+### HTTP
+```dvc
+    $ dvc run -d https://dvc.org/s3/examples/versioning/data.tgz \
```

Comment:
Let's use a less scary command as an example :slightly_smiling_face: 
```
dvc run -d https://example.com/data.txt \
             -o data.txt \
             wget https://example.com/data.txt -O data.txt
```

It is also more in-sync with the other examples.